file/lib/file_source_endpoint.ex

```diff
@@ -1,15 +1,18 @@
-defmodule Membrane.RTC.Engine.Support.FileSourceEndpoint do
-  @moduledoc false
-
-  # Endpoint that publishes data from a file.
-  # Starts publishing data on receiving `:start` message.
+defmodule Membrane.RTC.Engine.Endpoint.File do
+  @moduledoc """
+  An Endpoint responsible for publishing data from a file (Currently supports only OPUS and H264).
+  It starts publishing data on receiving `:start` message.
```

Comment:
I would mention here `start_sending` function instead. We should also mention about `:finished` message